components/home/pricing/pricing.tsx

```diff
@@ -0,0 +1,33 @@
+'use client'
+
+import Plans from './plans'
+
+interface PricingProps {
+  isLoggedIn: boolean
+}
+
+const Pricing = ({ isLoggedIn }: PricingProps) => {
+  return (
+    <div className="bg-white py-8 sm:py-24" id="pricing">
+      <div className="mx-auto max-w-7xl px-6 lg:px-8">
+        <div className="mx-auto max-w-4xl text-center">
+          <h2 className="text-base font-semibold leading-7 text-black">
+            Pricing
+          </h2>
+          <p className="mt-2 text-4xl font-bold tracking-tight text-gray-900 sm:text-5xl">
+            Pricing plans for teams of&nbsp;all&nbsp;sizes
+          </p>
+        </div>
+        <p className="mx-auto mt-6 max-w-2xl text-center text-lg leading-8 text-gray-600">
+          Distinctio et nulla eum soluta et neque labore quibusdam. Saepe et
+          quasi iusto modi velit ut non voluptas in. Explicabo id ut laborum.
+        </p>
```

Comment:
This could be moved to page.tsx for seo

Reply:
This component is called in home page as child